### DISCUSSION TITLE
Initialize the 🐼  library on Node.js 14.

### DISCUSSION THREAD

**Discussion by ryanpark** [chakra-ui/panda#1184] [General]
Hello 👋,

First of all, thank you for providing such a fantastic CSS library.
My current project is built on Node.js version 14, and unfortunately, I am unable to install it using Node.js version 16. 

This is causing an error when I attempt to run the command `npx panda init` resulting in the message: 

```"Unexpected token '||='```

Could you please suggest an alternative method for initializing Panda without the need to upgrade my Node.js version?  🙏 

Thank you 🐼

**Comment by omarkhatibgg** [comment]
if panda relies on new functionalities inside node 16 how you can avoid them? Unfortunately, there are no polyfills here.

The only way is to update your repository to use node 16 at least or use [nvn](https://github.com/nvm-sh/nvm) to switch between node versions

**Comment by ryanpark** [comment]
All good, ta
